armi/reactor/blueprints/__init__.py

```diff
@@ -508,6 +509,11 @@ def migrate(cls, inp: typing.TextIO):
                 inp = mig.apply()
         return inp
 
+    @classmethod
+    def load(cls, stream, roundTrip=False):
+        loader = RoundTripLoader if roundTrip else CLoader
+        return super().load(stream, Loader=loader)
+
```

Comment:
Could use an explanatory docstring about how cloader is faster but doesn't allow round tripping. Sidenote, are we sure the cloader still doesn't support roundtripping?

Reply:
1. Sure, I'll add a comment, that's fair.
2. Huh... I guess I assumed since it was call "Round Trip" Loader, and the other loader is WAY faster, there would be no reason to use it unless the other loader didn't do "round trip".  But I suppose I didn't verify that. hmm